Sources/UInt128/UInt128.swift

```diff
@@ -621,10 +636,33 @@ extension UInt128 : CustomStringConvertible {
         repeat {
             divmodResult = divmodResult.quotient.quotientAndRemainder(dividingBy: radix)
             index = characterPool.index(characterPool.startIndex, offsetBy: Int(truncatingIfNeeded: divmodResult.remainder.value.lowerBits))
-            result.insert(characterPool[index], at: result.startIndex)
+            result.append(characterPool[index])
         } while divmodResult.quotient > 0
-        return result
+        return String(result.reversed())
     }
+    
+//    internal func _valueToStringPrevious(radix: Int = 10, uppercase: Bool = true) -> String {
```

Comment:
With the test performed, are we good to remove this function? And if so, is there any value in keeping the new performance test module?

Reply:
I provided that for your comfort and convenience. You can blow it away if you like.

Pre-changes earlier in the week I was getting 0.0182-0.0184 averages. The latest was getting 0.0178. For my use case at +1000 it makes a difference.
For hex it could probably be done quicker than general algorithm, something low on my list as it currently works and I have other features to do first.

Its been my pleasure to contribute back.